projects/admin/src/app/record/detail-view/document-detail-view/related-resource/related-resource.component.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'admin-related-resource',
+  templateUrl: './related-resource.component.html'
+})
+export class RelatedResourceComponent {
```

Comment:
Is this component useful in other cases? If not it could be removed and integrated directly in ResourceComponent.

Reply:
Yes it's used in resource.component.html and in document-detail-view.component.html